### ISSUE TITLE
obs noise_scale_vec order error

### ISSUE THREAD

**Issue by BlackVegetab1e** [yusongmin1/Dreamwaq#2]
In function    _get_noise_scale_vec, noise order: 
        noise_vec[0: 3] = noise_scales.ang_vel * self.obs_scales.ang_vel   # ang vel
        noise_vec[3:6] = noise_scales.gravity         # euler x,y
        noise_vec[6:9] = 0. # commands
        noise_vec[9: 21] = noise_scales.dof_pos * self.obs_scales.dof_pos
        noise_vec[21: 33] = noise_scales.dof_vel * self.obs_scales.dof_vel
        noise_vec[33: 45] = 0.  # previous actions

but, in function    compute_observations(self), the true obs order is:
self.obs_buf = torch.cat((
            self.commands[:, :3] * self.commands_scale,
            self.base_ang_vel * self.obs_scales.ang_vel,
            self.projected_gravity,
            self.dof_err * self.obs_scales.dof_pos,
            self.dof_vel * self.obs_scales.dof_vel,
            self.actions
        ), dim=-1)

May be you chane the obs order to adape your robot, but forget changing the noise order.

**Comment by yusongmin1** [comment]
thanks
